<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>输入地址：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="forwardButton">
          <property name="text">
           <string>前进</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backButton">
          <property name="text">
           <string>后退</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refleshButton">
          <property name="text">
           <string>刷新</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWebView" name="webView">
        <property name="url">
         <url>
          <string>http://ele-dog.github.io/</string>
         </url>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1199</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="sourceAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="sourceAction">
   <property name="text">
    <string>查看源</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>forwardButton</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>977</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1066</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>1040</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refleshButton</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>reload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1146</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>1095</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
